### ISSUE TITLE
Concerns about conformance levels

### ISSUE THREAD

**Issue by myasonik** [w3c/silver#241]
_Spawned from a discussion in the [a11y community slack](https://twitter.com/a11yslack) between myself, @yatil, @yakimvanzuijlen, as well as others. I'll try to summarize as best I can but hope others can jump in if I misrepresented anything._

This concern is somewhat nebulous as it doesn't seem like the silver and gold conformance levels are strictly defined yet but based on them requiring "‘holistic tests’ such as assistive technology testing and usability testing with participants with disabilities” that does seem to leave the door open to shutting out small businesses and probably being difficult to smaller accessibility shops as well. 

Depending on what the final requirements might be, it would seem that requiring some sort of testing with actual people could be a requirement. While that's a great goal to have and a great thing to do, it's not very practical for small operations. 

A good quote from @yatil:
> The question is if it is a strategic good move to tell all smaller accessibility shops & clients that they cannot be conformant to a certain level of WCAG even if they have the same factual accessibility. Just because they cannot afford the certification cost.

And one from @yakimvanzuijlen:
> What matters is the end result. How you ended up there isn’t as important to the end user. You can do holistic testing badly and still have crappy accessibility... But you can wonder if it’s the purpose of WCAG to dictate the design process. That seems more like something that could be added in a non-normative supporting document. That could go more in-depth into best practices.

**Comment by hexabinaer** [comment]
Please help me understand what the core concern is:
a) costs/efforts involved in getting "actual people" to test
b) costs for certifcation

Regarding (a), I'd like to mention that usability tests can be driven at a really low cost level (reading recommendation: Steve Krug).

**Comment by yatil** [comment]
If it is for conformance, I assume it would need proper usability studies, but that is currently undefined. Testing with AT might be fairly straight-forward, but it is hard to know without any details on which AT to test and how to create a test plan.

Anyway those tests will make it harder to afford accessibility conformance validation. I think a) and b) are linked in that way.

**Comment by hexabinaer** [comment]
IMHO conformance rules are not very likely to keep pace with evolving technology. I had to overhaul a lot of theoretical knowledge when I started working with a blind colleage who, for example, uses VoiceOver. And I know from years of usability testing that it's a constant process of insights/learning.

Please don't get me wrong, I am fond of specifications and measurable results. My point here is: let's not argue against requiring usability tests by affected people. Let's instead offer them meaningful job opportunities. If the argument is about businesses (big or small) being able to afford the effort, my question is if our societies can afford to not take the effort ;-)

**Comment by yatil** [comment]
I’m not arguing against user testing in general. But there is a reason there are conformance levels to allow purchasers, for example, to differentiate.

If two products have the same usability and accessibility, but one can afford to be Gold conformant because they already are a big player (think Facebook) and the other one cannot, because it is not in reach financially (think micro.blog), that creates an imbalance that benefits big players and monopolies.

I have no issue with having a way to signal that the WCAG conformance has been augmented with user testing (or similar). But hinging the conformance level on it — especially without any details — seems to be at least discussion-worthy.

**Comment by hexabinaer** [comment]
Alright, that's a very valid point. 

On the other hand, when the conformance level does not specify details, how would that hinder small businesses to achieve higher conformance levels?

**Comment by yatil** [comment]
I assume it will specify details, it just doesn’t do this in the FPWD.

**Comment by cmuncharaz** [comment]
Yep, those silver and gold levels could create some elitism if they are out of reach of small players.

Also, another concern about the proposed new levels is that current accessibility regulations require conforming to WCAG 2.X Level AA. Since WCAG 3.0 is not backwards compatible with WCAG 2.X, that means that regulations would need to be updated at some point.

**Comment by yakimvanzuijlen** [comment]
When doing an accessibility audit, how would you judge if holistic testing has been done?

If there are no specific standards on how to do holistic testing it might mean some organisations will bring in testers to do the bare minimum to meet the requirement. Granted, there will always be cheaters.

To me this seems more like something for an Understand WCAG document (or the equivalent section in Silver). Holistic testing could be mentioned as a way to test methods.

**Comment by myasonik** [comment]
> When doing an accessibility audit, how would you judge if holistic testing has been done?

I assumed that holistic testing would be part of the audit. 

I think that's one of the problems with discussing this too much in depth is that there isn't a concrete proposal (that I've seen) for Silver and Gold so there are infinite of bad (and good) possibilities for what it can be. So we're spinning our wheels worrying about any of the bad possibilities but we don't know what's actually being considered.

**Comment by fstrr** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines \(WCAG\) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
